frontend_tests/node_tests/compose.js

```diff
@@ -2,6 +2,8 @@ global.patch_builtin('window', {
     bridge: false,
 });
 
+global.autosize = require('autosize/dist/autosize.js');
```

Comment:
Since it’s used as a locally required module and not as a global, we don’t need any of the `global.autosize` assignments in the tests—the tests donn’t need to change at all.

Reply:
Got it. Made the change.